lib/mayaUsd/ufe/UsdHierarchy.cpp

```diff
@@ -284,6 +331,10 @@ UsdHierarchy::insertChild(const Ufe::SceneItem::Ptr& child, const Ufe::SceneItem
 #ifdef UFE_V3_FEATURES_AVAILABLE
 Ufe::SceneItem::Ptr UsdHierarchy::createGroup(const Ufe::PathComponent& name) const
 {
+    // Changing the hierarchy of invalid items is not allowed.
+    if (!fItem->prim().IsActive())
+        return nullptr;
+
```

Comment:
Test is repeated a few places.  Would be best to put it inside UsdUndoCreateGroupCommand::create(), as the current code correctly deals with a nullptr cmd.

Reply:
The function is an override of a virtual function exposed by UFE, so it could get called from elsewhere than UsdUndoCreateGroupCommand::create(). Would it not be better to leave the checks here?